test/fast/Unit tests/nvm_get_node_from_pkg_json

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+die () { echo "$@" ; exit 1; }
+
+\. ../../../nvm.sh
+
+# TODO add more test cases
+TEST_CASES=(
+'{\n\t"engines": {\n\t\t"node":"1.1.1"\n\t}\n}"' '1.1.1'
```

Comment:
for test cases - let's make a dir containing a whole bunch of .json files, that are all example package.json files. That'll be easier to understand from a human perspective.